### PR TITLE
fix owner of /var/www/matomo/vendor/tecnickcom/tcpdf/fonts

### LINKED ISSUE #143: Generating E-Mail Reports Fails
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/crazy-max/docker-matomo/blob/master/.github/SUPPORT.md)

### I've found a bug and checked that ...

- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Hey!

We are trying to generate E-Mail reports. This currently fails in case of PDF reports due to some broken fonts(?)

### Expected behaviour

Report gets generated

### Actual behaviour

Generating report fails

### Steps to reproduce

1. Define a email report with target type PDF
2. Try to download the PDF
3. Generating fails

### Docker info

```text
Client: Docker Engine - Community
 Version:    24.0.7
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.11.2
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.21.0
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 6
  Running: 5
  Paused: 0
  Stopped: 1
 Images: 30
 Server Version: 24.0.7
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: journald
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 61f9fd88f79f081d64d6fa3bb1a0dc71ec870523
 runc version: v1.1.9-0-gccaecfc
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 6.1.0-11-amd64
 Operating System: Debian GNU/Linux 12 (bookworm)
 OSType: linux
 Architecture: x86_64
 CPUs: 8
 Total Memory: 15.62GiB
 Name: matomo
 ID: 38cc2464-ee67-4a7a-8ab3-381b56962e2c
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

### Docker Compose config

_No response_

### Logs

```text
WARNING ScheduledReports[2023-11-21 07:00:37 UTC] [0a025] /var/www/matomo/vendor/tecnickcom/tcpdf/include/tcpdf_static.php(1811): Warning - fopen(file:///var/www/matomo/vendor/tecnickcom/tcpdf/fonts/unifont.z): Failed to open stream: Permission denied - Matomo 4.15.0 - Please report this message in the Matomo forums: https://forum.matomo.org (please do a search first as it might have been reported already) [internal function]: Piwik\ErrorHandler::errorHandler(),#1/vendor/tecnickcom/tcpdf/include/tcpdf_static.php(1811),#2/vendor/tecnickcom/tcpdf/include/tcpdf_fonts.php(369),#3/core/ReportRenderer/Pdf.php(143),#4/plugins/ScheduledReports/API.php(564),[internal function]: Piwik\Plugins\ScheduledReports\API->generateReport(),#6/core/API/Proxy.php(244),#7/core/Context.php(28),#8/core/API/Proxy.php(335),#9/core/API/Request.php(267)
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] Uncaught exception: TypeError: fwrite(): Argument #1 ($stream) must be of type resource, bool given in /var/www/matomo/vendor/tecnickcom/tcpdf/include/tcpdf_fonts.php:370
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] Stack trace:
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #0 /var/www/matomo/vendor/tecnickcom/tcpdf/include/tcpdf_fonts.php(370): fwrite()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #1 /var/www/matomo/core/ReportRenderer/Pdf.php(143): TCPDF_FONTS::addTTFfont()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #2 /var/www/matomo/plugins/ScheduledReports/API.php(564): Piwik\ReportRenderer\Pdf->setLocale()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #3 [internal function]: Piwik\Plugins\ScheduledReports\API->generateReport()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #4 /var/www/matomo/core/API/Proxy.php(244): call_user_func_array()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #5 /var/www/matomo/core/Context.php(28): Piwik\API\Proxy->Piwik\API\{closure}()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #6 /var/www/matomo/core/API/Proxy.php(335): Piwik\Context::executeWithQueryParameters()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #7 /var/www/matomo/core/API/Request.php(267): Piwik\API\Proxy->call()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #8 /var/www/matomo/plugins/API/Controller.php(45): Piwik\API\Request->process()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #9 [internal function]: Piwik\Plugins\API\Controller->index()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #10 /var/www/matomo/core/FrontController.php(637): call_user_func_array()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #11 /var/www/matomo/core/FrontController.php(169): Piwik\FrontController->doDispatch()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #12 /var/www/matomo/core/dispatch.php(32): Piwik\FrontController->dispatch()
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #13 /var/www/matomo/index.php(25): require_once('...')
ERROR Piwik\FrontController[2023-11-21 07:00:38 UTC] [0a025] #14 {main}
```

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by fritterhoff**
Fixes #143

### CODE REVIEW

**Comment by crazy-max** [comment]
Thanks but I think it would be better to add it to https://github.com/crazy-max/docker-matomo/blob/186f7f46be42331b5128b75ac21419b0518641b6/rootfs/etc/cont-init.d/02-fix-perms.sh#L27

**Review by crazy-max — CHANGES_REQUESTED** [review]
Thanks but I think it would be better to add it to https://github.com/crazy-max/docker-matomo/blob/186f7f46be42331b5128b75ac21419b0518641b6/rootfs/etc/cont-init.d/02-fix-perms.sh#L27

**Comment by fritterhoff** [comment]
From my point of view, changing the folder permissions it self should be fine (without `-R`). I've changed the location :)

File: rootfs/etc/cont-init.d/02-fix-perms.sh [hunk 1]
```diff
@@ -32,4 +33,4 @@ chown -R matomo. \
   /var/run/nginx \
   /var/run/php-fpm \
   /var/www/matomo/config \
-  /var/www/matomo/js
+  /var/www/matomo/js
```
Comment (crazy-max) [inline rootfs/etc/cont-init.d/02-fix-perms.sh:36]: Remove extra space here

**Review by crazy-max — CHANGES_REQUESTED** [review]
One nit and also squash your commits please, thanks!

File: rootfs/etc/cont-init.d/02-fix-perms.sh (hunk 1 above)
Comment (fritterhoff) [reply rootfs/etc/cont-init.d/02-fix-perms.sh]: Done :)

**Review by crazy-max — APPROVED** [review]
Thanks

**Pull request merged**